tests/Unit/FeedXMLGenerationTest.php

```diff
@@ -6,7 +6,15 @@
 use \WC_Product_Variable;
 use \WC_Helper_Product;
 use \ReflectionClass;
+
 use Automattic\WooCommerce\Pinterest\ProductsXmlFeed;
+use Automattic\WooCommerce\Pinterest\Product\GoogleProductTaxonomy;
+use Automattic\WooCommerce\Pinterest\Product\GoogleCategorySearch;
+use Automattic\WooCommerce\Pinterest\Product\Attributes\Condition;
+use Automattic\WooCommerce\Pinterest\Product\Attributes\GoogleCategory;
+use Automattic\WooCommerce\Pinterest\Product\Attributes\AttributeManager;
+use ReflectionMethod;
```

Comment:
Not strictly necessary, but in other extensions we have kept the import statements in alphabetical order to provide some kind of organisation.